tests/pytorch_pfn_extras_tests/training_tests/extensions_tests/test_evaluator.py

```diff
@@ -188,3 +188,70 @@ def test_evaluator_progress_bar():
     reporter.add_observer('target', target)
     with reporter:
         evaluator.evaluate()
+
+
+# Code excerpts to test IgniteEvaluator
+class IgniteDummyModel(torch.nn.Module):
+    def __init__(self):
+        super(IgniteDummyModel, self).__init__()
+        self.count = 0.
+
+    def forward(self, *args):
+        ppe.reporting.report({'x': self.count}, self)
+        self.count += 1.
+        return 0.
+
+
+def create_dummy_evaluator(model):
+    from ignite.engine import Engine
+
+    def update_fn(engine, batch):
+        y_pred = torch.tensor(batch[1])
+        model()
+        # We return fake results for the reporters to
+        # and metrics to work
+        return (y_pred, y_pred)
+
+    evaluator = Engine(update_fn)
+    return evaluator
+
+
+class IgniteDataset(torch.utils.data.Dataset):
```

Comment:
I noticed shorter way to write dataset is:

```
n_data = 10
x = torch.randn((n_data , 2), requires_grad=True)
y = torch.randn((n_data , 2))
dataset = TensorDataset(y, t)
```

Reply:
Solved! thanks :D